index.html

```diff
@@ -15,17 +15,29 @@
   </head>
   <body>
       <header class="header-title">
-          <div class="navbar">
-              <a><p>James</p></a>
-              <nav class="navbar-list">
-                  <a href="#portfolio" > Portoflio </a>
-                  <a href="#about" > About </a>
-                  <a href="#contact" > Contact </a>
-              </nav>
-              <div class="icon">
-                <i class="fa fa-bars"></i>
-              </div>
-            </div>  
+        <div class="navbar">
+            <a><p>James</p></a>
+            <nav class="navbar-list" >
+                <i class="fa fa-times" id="closeMenu"></i>
```

Comment:
- [ ] You are almost there! However, there's a bug affecting the display of the (X) closing button, it's not visible:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-14 a la(s) 10 45 35 p  m](https://user-images.githubusercontent.com/83717285/173732652-e4168cb9-3f31-45de-83a6-76db816698a9.png)|![Captura de pantalla 2022-06-14 a la(s) 10 42 21 p  m](https://user-images.githubusercontent.com/83717285/173732497-af39fa3f-b842-4b1d-92b3-aeda234e2c0b.png)

- [ ] It would be great to fix the (X) button issue and try adding some top margin as in the Figma template.